linera-base/src/data_types.rs

```diff
@@ -466,6 +493,16 @@ impl Amount {
         Amount(10u128.pow(Amount::DECIMAL_PLACES as u32 - 3)).saturating_mul(millitokens)
     }
 
+    /// Returns an `Amount` corresponding to that many microtokens, or `Amount::MAX` if saturated.
+    pub fn from_micro(microtokens: u128) -> Amount {
```

Comment:
Should these names be plural? `from_{milli,micro,nano,atto}s` or even `from_{milli,micro,nano,atto}tokens` ?

Reply:
Perhaps native speakers can chime in @christos-h @Twey

Reply:
I agree, that's also what the standard library does. Standard library trumps native speakers!